package-lock.json

```diff
@@ -0,0 +1,17258 @@
+{
```

Comment:
Please remove this file. We use Yarn in this project, not NPM.

Reply:
Removed